app/server/components/services/github.mock.js

```diff
@@ -91,6 +92,7 @@ var mask = require('json-mask'),
         '2': {
             id: 2,
             name: 'zzz-permissive-repo-Test-Repo-1-pull',
+            description: 'zzz-permissive-repo-Test-Repo-1-pull',
```

Comment:
should these zzz teams be replaced entirely with "realistic" teams so it isn't confusing once they're all gone?